platform/ext/target/nordic_nrf/nrf5340/target_cfg.c

```diff
@@ -209,8 +209,8 @@ enum tfm_plat_err_t spu_periph_init_cfg(void)
 {
 	/* Peripheral configuration */
 	spu_peripheral_config_non_secure((uint32_t)NRF_FPU, false);
-	spu_peripheral_config_non_secure((uint32_t)NRF_OSCILLATORS, false);
-	spu_peripheral_config_non_secure((uint32_t)NRF_RESET, false);
+	spu_peripheral_config_non_secure((uint32_t)NRF_REGULATORS, false);
+	spu_peripheral_config_non_secure((uint32_t)NRF_CLOCK, false);
```

Comment:
OK @anangl so as far as I understand from the commit message, this is not wrong but just done for uniformity with nrf9160

Reply:
Yes, exactly, only for uniformity.